lib/chromic_pdf/api/pdf_options.ex

```diff
@@ -118,4 +118,23 @@ defmodule ChromicPDF.PDFOptions do
         {:ok, data}
     end
   end
+
+  def feed_merge_data_into_output(pdf_path, opts) do
+    case Keyword.get(opts, :output) do
+      path when is_binary(path) ->
+        File.cp!(pdf_path, path)
+        :ok
+
+      fun when is_function(fun, 1) ->
+        {:ok, fun.(pdf_path)}
+
+      nil ->
+        data =
+          pdf_path
+          |> File.read!()
+          |> Base.encode64()
+
+        {:ok, data}
+    end
+  end
```

Comment:
I think this is a copy of `PDFAOptions.feed_ghostscript_file_into_output/2`

Let's move this code to a different module, perhaps `ChromicPDF.Utils.feed_file_into_output/2` or so. Or into a new module in the `api/` directory, perhaps `OutputOption`?